subsys/net/ip/Kconfig.stats

```diff
@@ -69,7 +69,7 @@ config NET_STATISTICS_UDP
 config NET_STATISTICS_TCP
 	bool "TCP statistics"
 	depends on NET_TCP
-	default n
+	default y
```

Comment:
At this point we have already enabled general statistics, and if user has enabled TCP, he probably wants to see TCP stats. I see not much point disabling TCP stats by default here.